lib/slack-rtmapi/client.rb

```diff
@@ -97,6 +105,7 @@ def main_loop
       init
       loop do
         inner_loop
+        sleep 0.5
```

Comment:
Well, by using the Thread the reader is no longer blocking.
This is the idea because by taking the reader out of the base thread picture it can be interactive even when no data comes in.